ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx

```diff
@@ -85,6 +85,7 @@ const PostsList = ({ postData }: { postData: StreamResponse }) => {
   const { results: postResults } = postData?.data;
 
   if (!postResults) return null;
+  if (postResults.length === 0) return null;
```

Comment:
Do we need these checks if we're checking `{liveTextStream.content &&` in the `LivePageLayout` component?

Reply:
Possibly not. I have removed the `(!postResults)` check by updating the StreamResponse type, since the BFF should only return an array of Posts or an empty array.

I included the `(postResults.length === 0)` check to stop the PostList function passing an empty array to the PostItem function. I can remove the check without any visual impact/ errors appearing, though it seems logical to me to include this here.

Though do you think this check should happen in the LivePageLayout component?

Reply:
I think its ok to leave it. The `.map` function won't run if its an empty array, although it will render an empty `<div></div>`.

I'd just leave it, we'll be looking at each of these components more closely when the UX is ready anyway, so can always change the structure.